heron/packing/src/java/com/twitter/heron/packing/PackingUtils.java

```diff
@@ -160,4 +167,12 @@ public static boolean isValidInstance(Resource instanceResources,
     return currentContainers;
   }
 
+  public static long increaseBy(long value, int paddingPercentage) {
+    return value + (paddingPercentage * value) / 100;
+  }
+
+  public static double increaseBy(double value, int paddingPercentage) {
```

Comment:
Seems like we always round doubles. If so, maybe we should just do that as part of the method?